Library/Koala/Koala.swift

```diff
@@ -804,6 +850,8 @@ public final class Koala {
     self.track(event: "Canceled Share Sheet",
                properties: props)
 
+    guard shareContext.isLiveStreamContext != true else { return }
```

Comment:
we're doing these `guard`s cause we don't do any live stream tracking for deprecated events... cant wait to get rid of deprecated events :/